specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3258,6 +3258,76 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/blobUris": {
+      "get": {
+        "tags": [ "Reports" ],
+        "description": "Gets all Risky IP report URIs for the last 7 days.",
+        "x-ms-examples": {
+          "services_listAllRiskyIpDownloadReport": {
+            "$ref": "./examples/RiskyIpDownload.json"
+          }
+        },
+        "operationId": "services_listAllRiskyIpDownloadReport",

```

Comment:
operationId should not be camel-cased.

Reply:
See comment below.

Reply:
Hi Joel, Gaurav from ARM team has signed off, could you please sign off so that I can merge? I don't  have permissions to merge, am I correct in guessing that your signoff will cause merge to happen?